app/account/hooks/useUploadFile.ts

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react'
+import { createClient } from '@supabase/supabase-js'
+import { updateImageMetadataAction } from '../action/update-image-metadata'
+
+export const useUploadFile = () => {
```

Comment:
Is this also the same as the "useUploadFile" file under the trip? If so, it is a duplicate file. Could you merge them into one?

Reply:
It's a little different!  I had to tailor it to the account page.